scripts/zones/La_Vaule_[S]/mobs/Lobison.lua

```diff
@@ -14,7 +14,7 @@ end;
 -- onMobRoam Action
 -----------------------------------
 function onMobRoam(mob)
-    local spawnTime = mob:getLocalVar("transformTime");
+    local transformTime = mob:getLocalVar("transformTime");
```

Comment:
https://github.com/DarkstarProject/darkstar/blob/a0396e2e4cbbe93bd4949c9277f4e4b847b919d1/scripts/zones/Batallia_Downs_%5BS%5D/mobs/Gnole.lua
Would you mind making them match for consultancies sake?